File: WS_JAVA_REST_BanQuito/src/main/java/ec/edu/monster/model/Cliente.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.model;

/**
 *
 * @author danie
 */


import jakarta.persistence.*;
import java.time.LocalDate;

@Entity
@Table(name = "cliente")
public class Cliente {

    @Id
    @Column(length = 10)
    private String cedula;

    @Column(nullable = false, length = 100)
    private String nombre;

    @Column(name = "fecha_nacimiento")
    private LocalDate fechaNacimiento;

    @Column(name = "estado_civil", length = 10)
    private String estadoCivil; // SOLTERO, CASADO, etc.

    // Getters y setters
    // ...

    public String getCedula() {
        return cedula;
    }

    public void setCedula(String cedula) {
        this.cedula = cedula;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public LocalDate getFechaNacimiento() {
        return fechaNacimiento;
    }

    public void setFechaNacimiento(LocalDate fechaNacimiento) {
        this.fechaNacimiento = fechaNacimiento;
    }

    public String getEstadoCivil() {
        return estadoCivil;
    }

    public void setEstadoCivil(String estadoCivil) {
        this.estadoCivil = estadoCivil;
    }
}

File: PROYECTO/01.SERVIDOR/WS_DOTNET_SOAP_BanQuito/WS_DOTNET_SOAP_BanQuito/ec.edu.monster.model/Cliente.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WS_DOTNET_SOAP_BanQuito.ec.edu.monster.model

{
    [Table("cliente")]
    public class Cliente
    {
        [Key]
        [Column("cedula")]
        [StringLength(10)]
        public string Cedula { get; set; }

        [Required]
        [Column("nombre")]
        [StringLength(100)]
        public string Nombre { get; set; }

        [Column("fecha_nacimiento", TypeName = "date")]
        public DateTime? FechaNacimiento { get; set; }

        [Column("estado_civil")]
        [StringLength(10)]
        public string EstadoCivil { get; set; } // SOLTERO, CASADO, etc.

        // Navegación
        public virtual ICollection<Cuenta> Cuentas { get; set; }
        public virtual ICollection<Credito> Creditos { get; set; }
    }
}
